src/components/Form.js

```diff
@@ -1,89 +1,100 @@
+  };
+
+  const handleSearchQuery = (e) => {
+    e.preventDefault();
+    const url = forecast ? `https://api.weatherbit.io/v2.0/forecast/daily?city=${searchedQuery}&key=${fkey}` : `https://api.openweathermap.org/data/2.5/weather?q=${searchedQuery}&appid=${wkey}&units=metric`;
+
+    if (searchedQuery.trim().length > 2) {
+      if (hourly) {
+        onFetchHourlyByName(searchedQuery, fkey, wkey);
+      } else {
+        onFetchWeatherByName(forecast, url);
+      }
+
+      setSearchedQuery('');
+    } else {
+      onSetError({
+        msg: 'Min. 3 characters',
+      });
     }
+  };
 
-    const handleSearchQuery = (e) => {
-        e.preventDefault();
-        const url = forecast ? `https://api.weatherbit.io/v2.0/forecast/daily?city=${searchedQuery}&key=${f_key}` : `https://api.openweathermap.org/data/2.5/weather?q=${searchedQuery}&appid=${w_key}&units=metric`
+  const handleSearchQueryFromList = (query) => {
+    const url = forecast ? `https://api.weatherbit.io/v2.0/forecast/daily?city=${query}&key=${fkey}` : `https://api.openweathermap.org/data/2.5/weather?q=${query}&appid=${wkey}&units=metric`;
 
-        if(searchedQuery.trim().length > 2) {
-            if(hourly) {
-                onFetchHourlyByName(searchedQuery, f_key, w_key)
-            } else {
-                onFetchWeatherByName(forecast, url)
-            }
+    if (searchedQuery.trim().length > 2) {
+      if (hourly) {
+        onFetchHourlyByName(query, fkey, wkey);
+      } else {
+        onFetchWeatherByName(forecast, url);
+      }
 
-            setSearchedQuery('')
-        } else {
-            onSetError({
-                msg: 'Min. 3 characters'
-            })
-        }
+      setSearchedQuery('');
     }
+  };
 
-    const handleSearchQueryFromList = (query) => {
-        const url = forecast ? `https://api.weatherbit.io/v2.0/forecast/daily?city=${query}&key=${f_key}` :  `https://api.openweathermap.org/data/2.5/weather?q=${query}&appid=${w_key}&units=metric`
-
-        if(searchedQuery.trim().length > 2) {
-            if(hourly) {
-                onFetchHourlyByName(query, f_key, w_key)
-            } else {
-                onFetchWeatherByName(forecast, url)
-            }
-
-            setSearchedQuery('')
-        }
-    }
- 
-    return (
-        <FormContainer onSubmit={handleSearchQuery}>  
+  return (
+        <FormContainer onSubmit={handleSearchQuery}>
             <Input type='text' value={searchedQuery} placeholder='City' onInput={handleSetQuery} pattern="[A-Za-z\s]+" title="Please use only letters"/>
             <Submit type='submit'>
                 <img src={Glass} alt='glass' />
             </Submit>
-            {list ? <SearchList items={list} handleSearchQueryFromList={handleSearchQueryFromList} loading={loading} /> : null}
+            {list
+              ? <SearchList items={list}
+              handleSearchQueryFromList={handleSearchQueryFromList}
+              loading={loading} />
+              : null
```

Comment:
Looks like can use expression like `list && <SearchList... ` here

Reply:
Fixed that